src/engine/BMSkillTurbo.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * BMSkillTurbo: Code specific to the turbo die skill
+ *
+ * @author james
+ */
+
+/**
+ * This class contains code specific to the turbo die skill
+ */
+class BMSkillTurbo extends BMSkill {
+    /**
+     * An array containing the names of functions run by
+     * BMCanHaveSkill->run_hooks()
+     *
+     * @var array
+     */
+    public static $hooked_methods = array();
+
+
+    /**
+     * Description of skill
+     *
+     * @return string
+     */
+    protected static function get_description() {
+        return 'After your starting roll, you may change the size of a ' .
+               'Turbo Swing die or a Turbo Option die every time you roll it. ' .
+               'Decide on a size first that is valid for the Swing or Option '.
+               'type, then roll the new die as usual. If a Turbo Die is ' .
+               'rerolled by a player other than the player who has the Turbo ' .
+               'die (e.g., during a trip attack), then the size cannot be ' .
+               'changed.';
```

Comment:
Okay, so, why does this seem awkward to me?  Basically it just feels too wordy for the amount of information:
* why say "a Turbo Swing die or a Turbo Option die" rather than "a Turbo Swing or Option die"?
* "Decide on a size first that is valid for the Swing or Option type, then roll the new die as usual" - this feels both too verbose and insufficiently informative.  I feel like it sort of implies that you have to take an action to reroll the die after "deciding on a size".  Maybe something more like "Before each reroll, you will get the chance to select any valid size for the Swing or Option type"?
* "If a Turbo Die is rerolled by a player other than the player who has the Turbo die" - again, this seems awkward.  Do we really not have a more concise phrasing for "the player who owns a die"?